client/src/features/Event/New/components/EventCreateForm.tsx

```diff
@@ -61,7 +76,12 @@ export const EventCreateForm = () => {
         </Text>
 
         <Card>
-          <Text type="Body">기본 질문</Text>
+          <Flex justifyContent="space-between">
+            <Text type="Body">기본 질문</Text>
```

Comment:
p3.
으로 바꾸는게 디자인적으로 더 이쁠것 같아요~
```suggestion
            <Text type="Heading">기본 질문</Text>
```